airflow/hooks/postgres_hook.py

```diff
@@ -33,6 +34,14 @@ class PostgresHook(DbApiHook):
 
     Note: For Redshift, use keepalives_idle in the extra connection parameters
     and set it to less than 300 seconds.
+
+    Note: For AWS IAM authentication, use iam in the extra connection parameters
+    and set it to true. Leave the password field empty. This will use the the "aws_default"
+    connection to get the temporary token unless you override in extras.
+    extras example: {"iam":true,"aws_conn_id":"my_aws_conn"}
+    For Redshift, also add redshift in the extra connection parameters as an object with a
+    key, value of cluster-identifier and your cluster name.
+    extras example: {"iam":true, "redshift":{"cluster-identifier":"cluster-name"}}
```

Comment:
```suggestion
    extras example: ``{"iam":true, "redshift":{"cluster-identifier":"cluster-name"}}``
```

Reply:
Looks like you missed this one.

It would also be worth building the docs (`make -C docs html` if that works on windows) and checking how this renders.